torch/_torch_docs.py

```diff
@@ -1732,7 +1732,7 @@ def merge_dicts(*dicts):
 
 Args:
     input (Tensor): tensor to split.
-    indices_or_sections (Tensor, int or list or tuple of ints): See argument in :func:`torch.tensor_split`.
```

Comment:
Does not accept tensor
https://github.com/pytorch/pytorch/blob/8ad305f375bf20e88398f50b48425a3a7a80b685/aten/src/ATen/native/native_functions.yaml#L4407-L4411